added flag to disable crashlytics for debug builds

chaneylc opened the pull request:
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._



## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

<!-- 
Your release note should be written in clear and straightforward sentences. Most often, users aren't familiar with
the technical details of your PR, so consider what they need to know when you write your release note.

Some brief examples of release notes:
- Fixed a bug causing Field Book to crash when collecting categorical data.
- Added a new option to enable a sound when data is deleted.
- Modified the behavior trait drag and drop behavior.
-->

```release-note

```

trife commented:
@chaneylc Can we also change the `applicationIdSuffix` so that debug builds can be installed in parallel to releases? i.e. #741

trife commented:
@chaneylc can you finish this next week before the brapi hackathon so we can isolate any of those crashes?

chaneylc commented:
I wasn't able to filter by application id, but I was able to filter by custom keys. I added a key named "debug" with the boolean value of true for debug builds. You can filter this in the crashlytics search bar with "key: debug". Also added a custom key for release, similarly can be searched with "key: release"
